backend/wui/settings/testing.py

```diff
@@ -4,8 +4,49 @@
 
 DEBUG = True
 
-INSTALLED_APPS += [
-    "django_migration_linter",
+# Application definition
```

Comment:
I think the concatenation (`+=`)  on INSTALLED_APPS should work, the order there shouldn't matter... but we could double-check with apps' README (e.g. corsheaders install instructions).

Reply:
Okay got it.